content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,57 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers:
+
+**From a user to a controller**: 
+
+First the user signals to the controller that it should take ownership of a
+field, usually by setting some other field that the controller observes. The
+controller then claims ownership of the field by applying a configuration
+containing the field. This can be done in one of the following ways:
+
+- If the controller is designed to immediately set the value of the field, the
+controller should apply with force set to true. Once the user observes that the
+controller has taken ownership of the field, they can remove the field from
+their applied configuration to complete the transfer.
+- If the controller will not set the value of the field immediately, the user
+should genereate a unique, temporary, field manager name, and transfer ownership
+of the field to it. This transfer should be done by first setting the field to
```

Comment:
I'd move the "user to user" section above the user to controller section, so that here you can add:

```suggestion
of the field to it, similarly to the "user to user" instructions above. This transfer should be done by first setting the field to
```